sample_cgfs/rv32i_priv.cgf

```diff
@@ -1,21 +1,150 @@
 ecall:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ecall: 0
 
 ebreak:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ebreak: 0
+  
+misa:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*); def rvtest_mtrap_routine=True; def rvtest_strap_routine=True
+  csr_comb:
+    misa >> (xlen-2) == 0x01 : 0
+    (misa >> 30) & 0x3 == 0x3 : 0
+    misa & 0x30 == 0x30 : 0
+    misa & 0x04 == 0x01 : 0
+    misa & 0x08 == 0x01 : 0
+    misa & 0x03 == 0x01 : 0
+    misa & 0x05 == 0x01 : 0
+
+mvendorid:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*); def rvtest_mtrap_routine=True; def rvtest_strap_routine=True
+  mnemonics: 
+    csrrw: 0
+    csrrs: 0
+  csr_comb:
+    mvendorid & 0x80 == 0x80 : 0
+
+pmpcfg:
+  cond: check ISA:=regex(.*I.*Zicsr.*)
```

Comment:
It is possible for an implementation to have any of `0`,`16` or `64` pmp registers implemented on top of the fields being WARL themselves. Hence the tests and the coverpoints need to be enabled only if the pmp register is available and the particular scenario being tested is allowed. This comment is applicable for all the pmp coverpoints listed out.